internal/ratelimit/limiter_capacity_test.go

```diff
@@ -19,58 +19,60 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/jacobsa/ogletest"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 )
 
-func TestLimiterCapacity(t *testing.T) { RunTests(t) }
-
 ////////////////////////////////////////////////////////////////////////
 // Boilerplate
 ////////////////////////////////////////////////////////////////////////
 
 type LimiterCapacityTest struct {
+	suite.Suite
 }
 
-func init() { RegisterTestSuite(&LimiterCapacityTest{}) }
+func TestLimiterCapacitySuite(t *testing.T) {
+	suite.Run(t, new(LimiterCapacityTest))
+}
 
-func rateLessThanOrEqualToZero(rate float64) {
+func rateLessThanOrEqualToZero(t *testing.T, rate float64) {
 	_, err := ChooseLimiterCapacity(rate, 30)
 
-	expectedError := fmt.Errorf("Illegal rate: %f", rate)
+	expectedError := fmt.Sprintf("Illegal rate: %f", rate)
 
-	AssertEq(expectedError.Error(), err.Error())
+	assert.EqualError(t, err, expectedError)
 }
 
 func (t *LimiterCapacityTest) TestRateLessThanZero() {
 	var negativeRateHz float64 = -1
 
-	rateLessThanOrEqualToZero(negativeRateHz)
+	rateLessThanOrEqualToZero(t.T(), negativeRateHz)
 }
 
 func (t *LimiterCapacityTest) TestRateEqualToZero() {
 	var zeroRateHz float64 = 0
 
-	rateLessThanOrEqualToZero(zeroRateHz)
+	rateLessThanOrEqualToZero(t.T(), zeroRateHz)
 }
 
-func windowLessThanOrEqualToZero(window time.Duration) {
+func windowLessThanOrEqualToZero(t *testing.T, window time.Duration) {
 	_, err := ChooseLimiterCapacity(1, window)
 
-	expectedError := fmt.Errorf("Illegal window: %v", window)
+	expectedError := fmt.Sprintf("Illegal window: %v", window)
 
-	AssertEq(expectedError.Error(), err.Error())
+	assert.EqualError(t, err, expectedError)
```

Comment:
nit: (t, expected, actual)

Reply:
the order is correct: https://pkg.go.dev/github.com/stretchr/testify/assert#EqualError